demo/dismissable/index.html

```diff
@@ -0,0 +1,72 @@
+<html>
+
+<head>
+  <title>Dismiss element</title>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+  <style>
+    body {
+      font-family: Roboto, Helvetica, sans-serif;
+      max-width: 600px;
+      margin: 0px auto;
+      display: flex;
+      flex-direction: column;
+      background: whitesmoke;
+    }
+
+    virtual-scroller {
+      width: 100%;
+      height: 100%;
+      flex: 1;
+    }
+
+    contact-element {
+      background: white;
+      height: 80px;
+      padding: 0px 10px;
+    }
+  </style>
+</head>
+
+<body>
+
+  <virtual-scroller>
+    <template>
+      <dismissable-item>
+        <contact-element></contact-element>
+      </dismissable-item>
+    </template>
+  </virtual-scroller>
+
+  <script type="module">
+    import { ItemSource } from '../../virtual-scroller-element.js';
+    import './../contacts/contact-element.js';
+    import './dismissable-item.js';
+
+    const virtualScroller = document.querySelector('virtual-scroller');
+
+    const createElementFn = virtualScroller.createElement.bind(virtualScroller);
+    virtualScroller.createElement = (item, itemIndex) => {
```

Comment:
Are all current event listeners removed from an element when it is pushed to the element pool? Or that happens because it gets detached from the DOM?

I believe that adding the same function for the same event, should not duplicate the listener, so doing this should be fine.

Reply:
They're not removed, but as you noted we'd be adding the same function for the same event and get deduplication from the browser.

Alternatively, we could make `remove` event `bubble: true`, and have a single listener on virtualScroller - that should look even cleaner.
```js
// `remove` event bubbles
virtualScroller.addEventListener('remove', (event) => {
  const dismissable = event.target;
  myContacts.splice(dismissable.itemIndex, 1);
  virtualScroller.itemsChanged();
});
virtualScroller.updateElement = (element, contact, index) => {
  element.children[0].contact = contact;
  element.itemIndex = index;
};